source/material_model/entropy_model.cc

```diff
@@ -286,7 +331,7 @@ namespace aspect
                              "files located in the `data/' subdirectory of ASPECT.");
           prm.declare_entry ("Material file name", "material_table.txt",
                              Patterns::List (Patterns::Anything()),
-                             "The file name of the material data.");
+                             "The file name of the material data. The first one is for the background composition. ");
```

Comment:
Just to spell things out more clearly:
```suggestion
                             "The file name of the material data. The first material data file is intended for the background composition. ");
```